controllers/userController.js

```diff
@@ -52,6 +52,41 @@ module.exports = {
       });
   },
 
+  renderEdit: (req, res, next) => {
+    let userId = req.params.id;
+    User.findById(userId)
+      .then(user => {
+        res.render('user/edit', {
+          user: user
+        });
+      })
+      .catch(error => {
+        console.log(`Error fetching subscriber by ID:${error.message}`);
```

Comment:
i think here "User" instead of subscriber fits better (but this is a console.log so not important haha)